[2022-10-07 09:57:28,088] logging_util.py->write_log line:60 [INFO]预期结果：[{'equals': {'status_code': 200}}, {'contains': 'errcode'}]
[2022-10-07 09:57:28,095] logging_util.py->write_log line:60 [INFO]实际结果：{'errcode': 41004, 'errmsg': 'appsecret missing rid: 633f8787-73e9dda4-5a4582ee'}
[2022-10-07 09:57:28,102] logging_util.py->write_log line:60 [INFO]----------接口请求成功-------------
[2022-10-07 09:57:28,109] logging_util.py->write_log line:60 [INFO]----------接口请求结束-------------
[2022-10-07 09:57:28,118] logging_util.py->write_log line:60 [INFO]----------接口请求开始-------------
[2022-10-07 09:57:28,130] logging_util.py->write_log line:60 [INFO]接口名称：查询标签接口
[2022-10-07 09:57:28,137] logging_util.py->write_log line:60 [INFO]请求方发：get
[2022-10-07 09:57:28,144] logging_util.py->write_log line:60 [INFO]请求路径：https://api.weixin.qq.com/cgi-bin/tags/get?access_token
[2022-10-07 09:57:28,150] logging_util.py->write_log line:60 [INFO]请求参数：{'access_token': '61_FUQAt6D322ftApZv4tvuCoNWzD9OSPEroi29K4IhiEjKYa3X7XLVu950H4YZnY-Aj7NZSEuP3InWWD-VJOBpNhsEmkbz7r5FyeCTbWzYfR1Q84iJh0vi8PeLWRM8q19a-3qP-hTB0hxL9dOZZADbADAIJM'}
[2022-10-07 09:57:28,157] logging_util.py->write_log line:60 [INFO]文件上传：None
[2022-10-07 09:57:28,164] logging_util.py->write_log line:60 [INFO]请求头：{}
[2022-10-07 09:57:28,381] logging_util.py->write_log line:60 [INFO]预期结果：[{'equals': [{'status_code': 200}, {'sql': ''}]}, {'contains': 'tags'}]
[2022-10-07 09:57:28,388] logging_util.py->write_log line:60 [INFO]实际结果：{'tags': [{'id': 2, 'name': '星标组', 'count': 0}, {'id': 100, 'name': 'ms271387', 'count': 0}, {'id': 101, 'name': 'ms726424', 'count': 0}, {'id': 102, 'name': 'ms166704', 'count': 0}, {'id': 103, 'name': 'ms782539', 'count': 0}, {'id': 104, 'name': 'ms246465', 'count': 0}, {'id': 105, 'name': 'ms938595', 'count': 0}, {'id': 106, 'name': 'ms818515', 'count': 0}, {'id': 107, 'name': 'ms637042', 'count': 0}, {'id': 108, 'name': 'ms237049', 'count': 0}, {'id': 109, 'name': 'ms333399', 'count': 0}, {'id': 110, 'name': 'ms670805', 'count': 0}, {'id': 111, 'name': 'ms232139', 'count': 0}, {'id': 112, 'name': 'ms692329', 'count': 0}, {'id': 113, 'name': 'ms839440', 'count': 0}, {'id': 114, 'name': 'ms450932', 'count': 0}, {'id': 115, 'name': 'ms930749', 'count': 0}, {'id': 116, 'name': 'ms557405', 'count': 0}, {'id': 117, 'name': 'ms112540', 'count': 0}, {'id': 118, 'name': 'ms299799', 'count': 0}, {'id': 119, 'name': 'ms263315', 'count': 0}, {'id': 120, 'name': 'ms1', 'count': 0}, {'id': 121, 'name': 'ms12134', 'count': 0}, {'id': 122, 'name': 'ms7619', 'count': 0}, {'id': 123, 'name': 'ms5764', 'count': 0}, {'id': 124, 'name': 'ms4738', 'count': 0}, {'id': 125, 'name': 'ms8741', 'count': 0}, {'id': 126, 'name': 'ms3942', 'count': 0}, {'id': 127, 'name': 'ms8302', 'count': 0}, {'id': 128, 'name': 'ms8190', 'count': 0}, {'id': 129, 'name': 'ms7135', 'count': 0}, {'id': 130, 'name': 'ms4576', 'count': 0}, {'id': 131, 'name': 'ms1577', 'count': 0}, {'id': 132, 'name': 'ms7837', 'count': 0}, {'id': 133, 'name': 'ms2555', 'count': 0}, {'id': 134, 'name': 'ms5821', 'count': 0}, {'id': 135, 'name': 'ms6763', 'count': 0}, {'id': 136, 'name': 'ms4818', 'count': 0}, {'id': 137, 'name': 'ms2291', 'count': 0}, {'id': 138, 'name': 'ms8800', 'count': 0}, {'id': 139, 'name': 'ms6665', 'count': 0}, {'id': 140, 'name': 'ms8105', 'count': 0}, {'id': 141, 'name': 'ms8577', 'count': 0}, {'id': 142, 'name': 'ms3685', 'count': 0}, {'id': 143, 'name': 'ms4706', 'count': 0}, {'id': 144, 'name': 'ms9977', 'count': 0}, {'id': 145, 'name': 'ms9694', 'count': 0}, {'id': 146, 'name': 'ms2348', 'count': 0}, {'id': 147, 'name': 'ms3842', 'count': 0}, {'id': 148, 'name': 'ms2656', 'count': 0}, {'id': 149, 'name': 'ms1906', 'count': 0}, {'id': 150, 'name': 'ms7726', 'count': 0}, {'id': 151, 'name': 'ms7543', 'count': 0}, {'id': 152, 'name': 'ms1590', 'count': 0}, {'id': 153, 'name': 'ms2576', 'count': 0}, {'id': 154, 'name': 'ms7336', 'count': 0}, {'id': 155, 'name': 'ms5906', 'count': 0}, {'id': 156, 'name': 'ms6635', 'count': 0}, {'id': 157, 'name': 'ms1052', 'count': 0}, {'id': 158, 'name': 'ms9238', 'count': 0}, {'id': 159, 'name': 'ms6913', 'count': 0}, {'id': 160, 'name': 'ms2382', 'count': 0}, {'id': 161, 'name': 'ms7674', 'count': 0}, {'id': 162, 'name': 'ms2090', 'count': 0}, {'id': 163, 'name': 'ms6888', 'count': 0}, {'id': 164, 'name': 'ms8756', 'count': 0}, {'id': 165, 'name': 'ms1848', 'count': 0}, {'id': 166, 'name': 'ms1510', 'count': 0}, {'id': 167, 'name': 'ms1721', 'count': 0}, {'id': 168, 'name': 'ms5134', 'count': 0}, {'id': 169, 'name': 'ms3456', 'count': 0}, {'id': 170, 'name': 'ms7253', 'count': 0}, {'id': 171, 'name': 'ms3616', 'count': 0}, {'id': 172, 'name': 'ms9484', 'count': 0}, {'id': 173, 'name': 'ms5874', 'count': 0}, {'id': 174, 'name': 'ms2165', 'count': 0}, {'id': 175, 'name': 'ms1489', 'count': 0}, {'id': 176, 'name': 'ms4735', 'count': 0}, {'id': 177, 'name': 'ms5661', 'count': 0}, {'id': 178, 'name': 'ms6593', 'count': 0}, {'id': 179, 'name': 'ms8409', 'count': 0}, {'id': 180, 'name': 'ms1407', 'count': 0}, {'id': 181, 'name': 'ms5125', 'count': 0}, {'id': 182, 'name': 'ms9012', 'count': 0}, {'id': 183, 'name': 'ms1227', 'count': 0}, {'id': 184, 'name': 'ms9476', 'count': 0}, {'id': 185, 'name': 'ms6230', 'count': 0}, {'id': 186, 'name': 'ms1088', 'count': 0}, {'id': 187, 'name': 'ms2998', 'count': 0}, {'id': 188, 'name': 'ms2779', 'count': 0}, {'id': 189, 'name': 'ms1607', 'count': 0}, {'id': 190, 'name': 'ms7375', 'count': 0}, {'id': 191, 'name': 'ms4985', 'count': 0}, {'id': 192, 'name': 'ms8625', 'count': 0}, {'id': 193, 'name': 'ms1790', 'count': 0}, {'id': 194, 'name': 'ms5765', 'count': 0}, {'id': 195, 'name': 'ms3997', 'count': 0}, {'id': 196, 'name': 'ms3271', 'count': 0}, {'id': 197, 'name': 'ms6670', 'count': 0}, {'id': 198, 'name': 'ms2600', 'count': 0}, {'id': 199, 'name': 'ms2522', 'count': 0}]}
[2022-10-07 09:57:28,398] logging_util.py->error_log line:64 [ERROR]断言报错：异常信息：Traceback (most recent call last):
  File "C:\pycharm\common\requests_util.py", line 179, in validata_assert
    for assert_key, assert_value in dict(value).items():
ValueError: dictionary update sequence element #0 has length 1; 2 is required

[2022-10-07 09:57:28,405] logging_util.py->error_log line:64 [ERROR]热加载报错：异常信息：Traceback (most recent call last):
  File "C:\pycharm\common\requests_util.py", line 179, in validata_assert
    for assert_key, assert_value in dict(value).items():
ValueError: dictionary update sequence element #0 has length 1; 2 is required

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\pycharm\common\requests_util.py", line 120, in analysis_yaml
    self.validata_assert(caseinfo['validata'], return_data, status_code)
  File "C:\pycharm\common\requests_util.py", line 203, in validata_assert
    error_log("断言报错：异常信息：%s" % str(traceback.format_exc()))
  File "C:\pycharm\common\logging_util.py", line 65, in error_log
    raise Exception(log_message)
Exception: 断言报错：异常信息：Traceback (most recent call last):
  File "C:\pycharm\common\requests_util.py", line 179, in validata_assert
    for assert_key, assert_value in dict(value).items():
ValueError: dictionary update sequence element #0 has length 1; 2 is required


[2022-10-07 09:57:28,451] logging_util.py->write_log line:60 [INFO]----------接口请求开始-------------
[2022-10-07 09:57:28,459] logging_util.py->write_log line:60 [INFO]接口名称：新建标签
[2022-10-07 09:57:28,466] logging_util.py->write_log line:60 [INFO]请求方发：post
[2022-10-07 09:57:28,473] logging_util.py->write_log line:60 [INFO]请求路径：https://api.weixin.qq.com/cgi-bin/tags/create?access_token=61_FUQAt6D322ftApZv4tvuCoNWzD9OSPEroi29K4IhiEjKYa3X7XLVu950H4YZnY-Aj7NZSEuP3InWWD-VJOBpNhsEmkbz7r5FyeCTbWzYfR1Q84iJh0vi8PeLWRM8q19a-3qP-hTB0hxL9dOZZADbADAIJM
[2022-10-07 09:57:28,479] logging_util.py->write_log line:60 [INFO]请求参数：{'tag': {'name': 'ms4288'}}
[2022-10-07 09:57:28,486] logging_util.py->write_log line:60 [INFO]文件上传：None
[2022-10-07 09:57:28,492] logging_util.py->write_log line:60 [INFO]请求头：{}
[2022-10-07 09:57:28,736] logging_util.py->write_log line:60 [INFO]预期结果：[{'equals': {'status_code': 200}}, {'contains': 'tag'}]
[2022-10-07 09:57:28,742] logging_util.py->write_log line:60 [INFO]实际结果：{'errcode': 45056, 'errmsg': 'too many tag now, no need to add new hint: [4edEgBeVf-bi2iMA] rid: 633f8788-3425b88b-4c53e3f9'}
[2022-10-07 09:57:28,749] logging_util.py->write_log line:60 [INFO]----------接口请求成功-------------
[2022-10-07 09:57:28,755] logging_util.py->write_log line:60 [INFO]----------接口请求结束-------------
[2022-10-07 09:57:28,764] logging_util.py->write_log line:60 [INFO]----------接口请求开始-------------
[2022-10-07 09:57:28,773] logging_util.py->write_log line:60 [INFO]接口名称：文件上传
[2022-10-07 09:57:28,779] logging_util.py->write_log line:60 [INFO]请求方发：post
[2022-10-07 09:57:28,786] logging_util.py->write_log line:60 [INFO]请求路径：https://api.weixin.qq.com/cgi-bin/media/uploadimg?access_token=61_FUQAt6D322ftApZv4tvuCoNWzD9OSPEroi29K4IhiEjKYa3X7XLVu950H4YZnY-Aj7NZSEuP3InWWD-VJOBpNhsEmkbz7r5FyeCTbWzYfR1Q84iJh0vi8PeLWRM8q19a-3qP-hTB0hxL9dOZZADbADAIJM
[2022-10-07 09:57:28,792] logging_util.py->write_log line:60 [INFO]文件上传：None
[2022-10-07 09:57:28,799] logging_util.py->write_log line:60 [INFO]请求头：{}
